### DISCUSSION TITLE
Manipulate String n8n

### DISCUSSION THREAD

**Discussion by hasanbasri1993** [devlikeapro/waha#571] [General]
i'm triying to use n8n and i set the trigger 
when the body is contain CHANNEL_
then send back agai but i nneed replace CHANNEL_
i try like this bu nothing happen using Code  Block
`return $input.payload.body.replace("CHANNEL_", "");`

Thanks u

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! It's more n8n related question, not the project.
Here's what chatgpt answered (I haven't tried it tho)
```js
// Accessing input data (adjust this part if your structure is different)
const payload = $input.item.json; 

// Check if the body field exists and is a string
if (payload.body && typeof payload.body === 'string') {
  // Replacing all occurrences of "CHANNEL_" with an empty string
  const modifiedBody = payload.body.replace(/CHANNEL_/g, "");
  
  // Return the modified body
  return [{ json: { body: modifiedBody } }];
} else {
  throw new Error('The input body is not a string or does not exist');
}
```

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by hasanbasri1993** [reply]
thank u

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
